UICatalog/Scenarios/TableEditor.cs

```diff
@@ -14,7 +14,6 @@ namespace UICatalog.Scenarios;
 [ScenarioCategory ("Controls")]
 [ScenarioCategory ("Dialogs")]
 [ScenarioCategory ("Text and Formatting")]
-[ScenarioCategory ("Overlapped")]
 public class TableEditor : Scenario
```

Comment:
Sometimes after open this scenario and navigating through the lines when I press escape to exit the main UICatalog window isn't redrawn totally but only half of the window. This happening with the window maximized. I have to click to redraw it.

Reply:
Sometimes is when I open the scenario it draw well and then it stay with a black screen.

Reply:
I'll take a look.. Thanks.

Reply:
Can't repro.